src/Couchbase.Lite.Shared/Sync/WebSocketWrapper.cs

```diff
@@ -573,6 +573,7 @@ private async void PerformWrite()
                         await stream.WriteAsync(nextData, 0, nextData.Length, cancelSource.Token).ConfigureAwait(false);
                     } catch (Exception e) {
                         DidClose(e);
+                        await NetworkStream?.FlushAsync(cancelSource.Token);
```

Comment:
575 is going to set `NetworkStream` to `null` so it seems like this line is not doing anything.